sbi/inference/trainers/nre/nre_base.py

```diff
@@ -117,6 +117,8 @@ def append_simulations(
         Returns:
             NeuralInference object (returned so that this function is chainable).
         """
+        if exclude_invalid_x:
+            raise ValueError("The 'exclude_invalid_x' parameter must be False.")
```

Comment:
same as above: please change this to a warning and adapt the warn message to what we are saying in the docstring, e.g., that NRE will give systematically biased results when used this way.